### PR TITLE
Adding canInviteCollaborator to BOXItem.

### PR DESCRIPTION
**Pull request by scottliubox**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @scottliubox has signed the CLA. Thanks for the pull request!

File: BoxContentSDK/BoxContentSDK/Models/BOXItem.h [hunk 1]
```diff
@@ -183,6 +183,13 @@
 @property (nonatomic, readwrite, assign) BOXAPIBoolean canDelete;
 
 /**
+ *  Indicates permission for the current user can invite users into the folder as collaborators.
```
Comment (boxtcarpel) [inline BoxContentSDK/BoxContentSDK/Models/BOXItem.h]: update the comment, now it's not just about folder collaborators.

File: BoxContentSDK/BoxContentSDK/Models/BOXItem.h [hunk 2]
```diff
@@ -183,6 +183,13 @@
...
+ *  Warning: By default, the Box API does not return this value, and it will be nil.
+ *  You must request it by setting the "fields" of the request.
```
Comment (boxtcarpel) [inline BoxContentSDK/BoxContentSDK/Models/BOXItem.h]: Should add a note that it is currently not supported for box bookmarks.

**Review by boxtcarpel — APPROVED** [review]
A couple of comments

**Pull request merged**